app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -3938,11 +3938,12 @@ class PatientRepositoryAndroidTest {
         .map { it.uuid }
 
     //then
-    assertThat(searchResults).containsExactly(
-        patient2WithCurrentFacilityAsAssignedFacility,
-        patientWithCurrentFacilityAsAssignedFacility,
-        patientWithAnotherFacilityAsAssignedFacility
-    )
+    assertThat(searchResults)
+        .containsExactly(
+            patientWithCurrentFacilityAsAssignedFacility,
+            patient2WithCurrentFacilityAsAssignedFacility,
+            patientWithAnotherFacilityAsAssignedFacility
+        ).inOrder()
```

Comment:
Correct me if I am wrong, will `contains` doesn't verify the order but passes even if there are more items than expected, `containsExactly` verifies that it has the same items as the expected but doesn't verify the order right?

Reply:
Yup, that is correct.

Reply:
The clue is in the assertion name itself, `contains`.